hack/.ci/run-e2e-gke.sh

```diff
@@ -140,60 +142,132 @@ kubectl_create -n default -f - <<EOF
-kubectl -n scylla-manager rollout status deployment/scylla-manager-controller
+kubectl -n=scylla-manager patch --field-manager="${field_manager}" deployment/scylla-manager-controller --type=json -p='[{"op": "add", "path": "/spec/template/spec/containers/0/args/-", "value": "--loglevel=4"}]'
+kubectl -n=scylla-manager rollout status deployment/scylla-manager-controller
 
 ingress_class_name='haproxy'
 ingress_custom_annotations='haproxy.org/ssl-passthrough=true'
-ingress_controller_address="$( kubectl -n haproxy-ingress get svc haproxy-ingress --template='{{ .spec.clusterIP }}' ):9142"
+ingress_controller_address="$( kubectl -n=haproxy-ingress get svc haproxy-ingress --template='{{ .spec.clusterIP }}' ):9142"
 
-kubectl create -n e2e pdb my-pdb --selector='app=e2e' --min-available=1 --dry-run=client -o yaml | kubectl_create -f -
+kubectl create -n=e2e pdb my-pdb --selector='app=e2e' --min-available=1 --dry-run=client -o=yaml | kubectl_create -f=-
 
-kubectl -n e2e run --restart=Never --image="${SO_IMAGE}" --labels='app=e2e' --command=true e2e -- bash -euExo pipefail -O inherit_errexit -c "function wait-for-artifacts { touch /tmp/done && until [[ -f '/tmp/exit' ]]; do sleep 1; done } && trap wait-for-artifacts EXIT && mkdir /tmp/artifacts && scylla-operator-tests run '${SO_SUITE}' --loglevel=2 --color=false --artifacts-dir=/tmp/artifacts --feature-gates='${SCYLLA_OPERATOR_FEATURE_GATES}' --ingress-controller-address='${ingress_controller_address}' --ingress-controller-ingress-class-name='${ingress_class_name}' --ingress-controller-custom-annotations='${ingress_custom_annotations}' --scyllacluster-node-service-type='${SO_SCYLLACLUSTER_NODE_SERVICE_TYPE}' --scyllacluster-nodes-broadcast-address-type='${SO_SCYLLACLUSTER_NODES_BROADCAST_ADDRESS_TYPE}' --scyllacluster-clients-broadcast-address-type='${SO_SCYLLACLUSTER_CLIENTS_BROADCAST_ADDRESS_TYPE}'"
-kubectl -n e2e wait --for=condition=Ready pod/e2e
+gcs_sa_in_container_path=""
+if [[ -n "${SO_GCS_SERVICE_ACCOUNT_CREDENTIALS_PATH+x}" ]]; then
+  gcs_sa_in_container_path=/var/run/secrets/gcs-service-account-credentials/gcs-service-account.json
+  kubectl create -n=e2e secret generic gcs-service-account-credentials --from-file="${SO_GCS_SERVICE_ACCOUNT_CREDENTIALS_PATH}" --dry-run=client -o=yaml | kubectl_create -f=-
+else
+  kubectl create -n=e2e secret generic gcs-service-account-credentials --dry-run=client -o=yaml | kubectl_create -f=-
+fi
+
+kubectl_create -n=e2e -f=- <<EOF
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: e2e
+  name: e2e
+spec:
+  restartPolicy: Never
+  containers:
+  - name: wait-for-artifacts
+    command:
+    - bash
+    - -euExo
+    - pipefail
+    - -O
+    - inherit_errexit
+    - -c
+    args:
+    - |
+      function wait-for-artifacts {
+       touch /tmp/done
+       until [[ -f '/tmp/exit' ]]; do
+        sleep 1
+       done
```

Comment:
nit: use 2 spaces for indentation (it has 1 space only)

Reply:
done